core/src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -742,12 +747,16 @@ import akka.stream.scaladsl.Source
                 None,
                 settings.journalSettings.readProfile,
                 "asyncReplayMessages",
-                extractor = Extractors.taggedPersistentRepr(eventDeserializer, serialization))
+                extractor = Extractors.taggedPersistentRepr(eventDeserializer, serialization),
+                ec)
               .mapAsync(1)(tr.sendMissingTagWrite(tp))
```

Comment:
This is likely one of the cases where we also need to set the dispatcher on the inner flow (because of bug where `futureSource` does not pass attributes to nested flow before running it correctly)

Reply:
I have added my println testing again
https://github.com/akka/akka-persistence-cassandra/pull/889/commits/a5817a67b4899d303c739d9cb552c25262edb1b2

When running akka.persistence.cassandra.journal.CassandraIntegrationSpec it's all on `journal-dispatcher`.

I have tried with both Akka 2.6.9 and 2.6.14.

It seems to work fine. Do you see any mistakes in my println?